float64 latitude
float64 longitude
uint16 altitude
